frontend/src/app/app.component.spec.ts

```diff
@@ -2,8 +2,10 @@ import { TestBed } from '@angular/core/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { AppComponent } from './app.component';
 
-describe('AppComponent', () => {
-  beforeEach(async () => {
+describe('AppComponent', () =>
+{
```

Comment:
This is wrong formatting. Curly brace should start on the previous line. Fix here and everywhere